ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -804,6 +859,44 @@ void OwInterface::guardedMoveAction (double x, double y, double z,
     guarded_move_done_cb<GuardedMove, ow_lander::GuardedMoveResultConstPtr>);
 }
 
+std::vector<double> OwInterface::identifySampleLocation (int num_images, std::string filter_type, int id)
```

Comment:
const std::string& is better, and idiomatic.  

Style comment: for more readability and shorter lines, you could add a `using std::string;` at the top of the file, and just say `string` here and everywhere else in this file.